components/sysutils/slurm/patches/11-getgrouplist.patch

```diff
@@ -0,0 +1,214 @@
+--- ./src/slurmd/slurmd/req.c.orig	2016-05-05 22:21:22.550661771 +0200
```

Comment:
My understanding was that adding BSD-licensed code patches (or any permissive copyleft) to a GPL licensed code is not an issue while the converse is not possible: GPL code requires that the overall license is GPL.